getting-started.md

```diff
@@ -1,35 +1,45 @@
 # Getting Started
 
-## Creating Account
+### Creating Account
 
-You will need the Exogress account to use the service. Please, visit [https://app.exogress.com](https://app.exogress.com/) and create a new account.
+Visit [https://app.exogress.com](https://app.exogress.com/) to create a new account and follow the instructions. If you signed up using email, make sure you have activated your account.
 
-## Creating Project
+### Choose your account and subdomain names
 
-Project is the place where your configurations live in. You will be prompted to create your first project once your account is ready. Later, you'll be able to add more projects.
+You will be prompted to provide your account name. This will be associated with your third-level domain name in Exogress, and couldn't be changed.
+Next, we ask you to specify a fourth-level subdomain - this will be the first default url. These domains are free and you could always add your own domain at any time later.
 
-## Creating Access Token
+### Creating Project
 
-Click the `Access Tokens` section in the sidebar. Write down the access token name and press `Create Access Token`. You will need two tokens:
+Project is a place your configuration lives in, and you could add and delete projects at any time.
+Navigate to **Projects** and click `+Add project`. Give Project a name and click `Save`
+
+### Creating Access Token
+
+Access tokens are required for spawning Exogress client.
+Click the `Access Tokens` section in the sidebar. Provide access token name and click `Create Access Token`. You will need two tokens:
 
 - `ACCESS_KEY_ID`
 - `SECRET_ACCESS_KEY`
 
 Make sure you save the secret token to a secure place; it will be impossible to retrieve it later.
 
-## Adding your Domain
+### Adding your Domain
+
+Exogress allows to create a subdomain under the `your_account.exg.link` or to add your own domain.
 
-By default, Exogress allows to create subdomain under the `exg.link` or to add your own domain.
+Visit `Domain` section in the sidebar, click `Add Domain in `your_account.exg.link`` and enter desired hostname under your fourth-level domain, then click `Save`. Exogress will take care of issuing TLS certificates for you. Wait for the certificate status to become `ready` as you won't be able to use the service until that.
 
-Visit the `Domain` section in the sidebar and enter the desired domain name, and click `Create`. Exogress will take care of issuing TLS certificates for you. Wait for the certificate state to become `ready`. You will be unable to use the service until that.
+To add your own domain, navigate to the "Custom domains" tab and click `Add Custom Domain`. There, provide your custom domain name and strict transport security expiration period in seconds (recommended is 31536000).
```

Comment:
Mention that the user needs to create the DNS record and wait some time for changes to propagate.